src/redux/actions.js

```diff
@@ -0,0 +1,6 @@
+export function dummyFunction(dummy) {
+  return {
+    type: 'DUMMY',
+    dummy
```

Comment:
Wiem, że to przykład, ale na przyszłość jestem za tym żeby trzymać się konwencji z nazwaniem danych jakie niesie ze sobą akcja 'payload'. Czyli w tym przypadku return {type: 'DUMMY', payload: {dummy}}. 
To po prostu konwencja którą zazwyczaj się stosuję. Nawet było o tym na ostatnim React Meetupie. 
Jeśli jesteś za to poproszę o łapkę w górę 👍